app/models/post_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) do
+    User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
```

Comment:
- [ ] You have done a good job working on validations and model specs 👍, however, you are not using a good directory for your `_spec.rb`. Kindly, make sure you use this example:

````
spec/models / - like_spec.rb
               - comment_spec.rb
               - post_spec.rb
               - user_spec.rb
````

![Screenshot 2023-08-16 012056](https://github.com/SergioPeralta22/MV-rails-blog-app/assets/57408419/a6fb783e-c9a5-40ef-bac9-134278046164)


Kindly fix it 😄

Reply:
Hahaha, my bad